src/Microsoft.Graph/Microsoft.Graph.Beta.csproj

```
@@ -26,7 +26,9 @@
 - https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/blob/master/CHANGELOG.md
     </PackageReleaseNotes>
     <!-- edit this value to change the current major.minor.patch version -->
+     <!-- x-release-please-start-version -->
     <VersionPrefix>5.78.0</VersionPrefix>
```

Something to think about here/confirm, if the automation will add the `-preview` suffix here, we may need to change this tag to something like this and drop line 37(`VersionSuffix` tag).
```suggestion
    <Version>5.78.0-preview</Version>
```

Otherwise, we may end up with a `5.78.0-preview-preview` package name...